src/imitation_cli/algorithm_configurations/airl.py

```diff
@@ -0,0 +1,33 @@
+"""Config for AIRL."""
+import dataclasses
+from typing import Optional
+
+from omegaconf import MISSING
+
+from imitation_cli.utils import environment as environment_cfg
+from imitation_cli.utils import (
+    optimizer_class,
+    reward_network,
+    rl_algorithm,
+    trajectories,
+)
+
+
+@dataclasses.dataclass
+class Config:
```

Comment:
nit: this is a bit confusing—won't users of this class always have to import it as `AIRLConfig` or something that disambiguates between the other configs?

Reply:
We have the convention to not directly import classes, so in a different file we would write:
```python
from imitation_cli.algorithm_configurations import airl
...
my_conf = airl.Config()
```